testsuite/features/core/proxy_branch_network.feature

```diff
@@ -35,6 +34,7 @@ Feature: Setup Uyuni for Retail branch network
     When I refresh the metadata for "server"
     When I install pattern "suma_retail" on this "server"
     And I wait for "patterns-suma_retail" to be installed on "server"
+    And I synchronize all Salt dynamic modules on "proxy"
```

Comment:
don't we also need this fix for `features/build_validation/retail/proxy_branch_network.feature`?

Reply:
it is already there